tools/appliance/systemvmtemplate/scripts/install_systemvm_packages.sh

```diff
@@ -78,12 +78,7 @@ function install_packages() {
   apt-get clean
   apt-get autoclean
 
-  #32 bit architecture support:: not required for 32 bit template
-  if [ "${arch}" != "i386" ]; then
-    dpkg --add-architecture i386
-    apt-get update
-    ${apt_get} install links:i386 libuuid1:i386 libc6:i386
-  fi
+  ${apt_get} install links
```

Comment:
The vhd-util we bundle with systemvm.iso/cloudstack is used by ssvm, and needs the i386 libs:
ldd ./scripts/vm/hypervisor/xenserver/vhd-util
	linux-gate.so.1 =>  (0xf7f23000)
	libuuid.so.1 => /lib/i386-linux-gnu/libuuid.so.1 (0xf7edb000)
	libc.so.6 => /lib/i386-linux-gnu/libc.so.6 (0xf7d05000)
	/lib/ld-linux.so.2 (0xf7f25000)

Reply:
Does it need the 32 bits library? Or, is it a matter of creating a symbolic link to the files of the 64 bits library?

Reply:
Yes, see the ldd output. Either we'll need libuuid:i386 or rebuild/ship a 64-bit vhd-util binary.

Reply:
I mean, can't we simply do the following?
```
mkdir /lib/x86_64-linux-gnu/
ln -s /lib/x86_64-linux-gnu/libuuid.so.1 /lib/i386-linux-gnu/libuuid.so.1
ln -s /lib/x86_64-linux-gnu/libc.so.6 /lib/i386-linux-gnu/libc.so.6
```

The libraries are already in the 64bits system when we install the "links" for 64bits. The problem is that they are in a different path that `vhd-util` does not know.

If we create the symbolic links, we would have the following then:
```
root@debian:~# ls -lah /lib/i386-linux-gnu/
lrwxrwxrwx  1 root root   31 Jun 14 08:25 libc.so.6 -> /lib/x86_64-linux-gnu/libc.so.6
lrwxrwxrwx  1 root root   34 Jun 14 08:24 libuuid.so.1 -> /lib/x86_64-linux-gnu/libuuid.so.1
```

Would this work?

Reply:
@rhtyd if we have the proper version to rebuild that would be best. In any way teh cloudstack-common package doesn't need it, only the systemvm template.
we can either
- ammend the commit that removes the 32 dependencies to allow for glibc and libuuid, or
- rebuild vhdutil
What ever is easiest will do for this release, i think

Reply:
@DaanHoogland I can try to build vhd-util x86_64 binary for linux, but we can do that for the future 4.11.2.0 release. The easiest would be to continue to install the 32bit dependencies in systemvm like we've done in the current 4.11.1 systemvmtemplate.

Reply:
@DaanHoogland Based on the last comment, can you remove the changes to the systemvm template build script? Once we have the 64-bit vhd-util we can remove the changes from this file.

Reply:
will do, and then of course test if this fixes the centos7 dependencies at least.